<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CodeTaeed</class>
 <widget class="QMainWindow" name="CodeTaeed">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 231, 184);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="C_F">
      <property name="maximumSize">
       <size>
        <width>450</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 247, 0,70);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QFrame" name="C_F_MainText">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="C_LabalMainText">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>60</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="text">
             <string>لطفا کد تایید را وارد کنید</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QFrame" name="C_F_EnterCode">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="2" column="1">
           <widget class="QLabel" name="C_CaptionLabel">
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QFrame" name="frame_2">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_6">
             <item row="1" column="0">
              <widget class="QPushButton" name="C_Button">
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="text">
                <string>تایید</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="0" rowspan="2" colspan="2">
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="0" column="0">
              <widget class="QSplitter" name="splitter_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <widget class="QSplitter" name="splitter">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <widget class="QLineEdit" name="C_LineEdit1EnterCode">
                 <property name="minimumSize">
                  <size>
                   <width>55</width>
                   <height>55</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>55</width>
                   <height>55</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>20</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="maxLength">
                  <number>1</number>
                 </property>
                </widget>
                <widget class="QLineEdit" name="C_LineEdit2EnterCode">
                 <property name="minimumSize">
                  <size>
                   <width>55</width>
                   <height>55</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>55</width>
                   <height>55</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>20</pointsize>
                  </font>
                 </property>
                 <property name="maxLength">
                  <number>1</number>
                 </property>
                </widget>
                <widget class="QLineEdit" name="C_LineEdit3EnterCode">
                 <property name="minimumSize">
                  <size>
                   <width>55</width>
                   <height>55</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>55</width>
                   <height>55</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>20</pointsize>
                  </font>
                 </property>
                 <property name="maxLength">
                  <number>1</number>
                 </property>
                </widget>
                <widget class="QLineEdit" name="C_LineEdit4EnterCode">
                 <property name="minimumSize">
                  <size>
                   <width>55</width>
                   <height>55</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>55</width>
                   <height>55</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>20</pointsize>
                  </font>
                 </property>
                 <property name="maxLength">
                  <number>1</number>
                 </property>
                </widget>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>584</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>C_LineEdit1EnterCode</tabstop>
  <tabstop>C_LineEdit2EnterCode</tabstop>
  <tabstop>C_LineEdit3EnterCode</tabstop>
  <tabstop>C_LineEdit4EnterCode</tabstop>
  <tabstop>C_Button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
